napari/layers/image/image.py

```diff
@@ -273,29 +209,16 @@ def __init__(
                 trans._('Image data must have at least 2 dimensions.')
             )
 
-        # Determine if data is a multiscale
+        # Deter
```

Comment:
? 🤔

Reply:
I was moving chak multiscaleand and bug happens